docs/command/mongocli-cloud-manager-automation-watch.txt

```diff
@@ -14,6 +14,11 @@ mongocli cloud-manager automation watch
 
 Watch for automation changes to be completed.
 
+This command checks the automation goal version periodically until it's reached.
```

Comment:
I don't know what is goal version (at least we don't talk about it in the docs in this way). So, is it possible to use something else here? Sorry I don't have suggestions, but the team must have them.

Reply:
Could we change this to "This command periodically checks whether the MongoDB version matches the goal version set in automation, until the goal version is reached."

Reply:
From my side, the proposed one above sounds good. @gssbzn WDYT?

Reply:
this is not about the mdb version, docs on the api https://docs.opsmanager.mongodb.com/current/reference/api/automation-status/

`goalVersion` is a field there, first on the example, we do some checks in a couple of places to make sure the automation has reached that number

Reply:
Oh I see. So can we then use goalVersion in the annotation, so it is clear it is a parameter? Then we can keep Bianca's original annotation, just with this goalVersion.

Reply:
users' don't provide it, it's all handled internally, automation exposes both "current" and "goal" and we make sure they are equal, goal is incremented every time a new change is published to the automation config , current increment every-time the changes are done

Reply:
thanks for the explanation, I think it makes sense to keep this now. With the "goal version" sentence.